src/SFA.DAS.AANHub.Application.UnitTests/CalendarEvents/Queries/GetCalendarEventByIdQueryHandlerTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.CalendarEvents.Queries.GetCalendarEvent;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.CalendarEvents.Queries;
+
+public class GetCalendarEventByIdQueryHandlerTests
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task Handle_ReturnsNull_IfCalendarEventId_DoesNotExist(
+        [Frozen] Mock<ICalendarEventsReadRepository> calendarEventsReadRepositoryMock,
+        GetCalendarEventByIdQueryHandler sut,
+        CalendarEvent nonExistentCalendarEvent,
+        Member member)
+    {
+        calendarEventsReadRepositoryMock.Setup(c => c.GetCalendarEvent(nonExistentCalendarEvent.Id))
+                                        .ReturnsAsync(() => null);
+
+        var result = await sut.Handle(new GetCalendarEventByIdQuery(nonExistentCalendarEvent.Id, member.Id), new CancellationToken());
+
+        result.Result.Should().BeNull();
+    }
+
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task Handle_ReturnsCalendarEvent_IfCalendarEventId_AndRequestedUserId_Exist(
```

Comment:
Naming convention is <Action>_<Condition>_<Result> as explained in the Wow document. Suggest rename to `Handle_CalendarEventFound_ReturnsCalendarEvent`